packages/react-link/src/components/Link/Link.types.ts

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import { ComponentProps } from '@fluentui/react-utils';
+import { ColorTokens, ColorTokenSet, FontTokens, RecursivePartial } from '@fluentui/theme';
```

Comment:
This types are not coming from converged package.

Reply:
@fluentui/react-theme

Reply:
Removed